app/views/posts/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<% @posts.each do |post| %>
+<% @posts.reverse.each do |post| %>
```

Comment:
This does 'work', but only because our IDs happen to go up in sequence right now. This might not always be true. Instead of this, I'd use this code:

```ruby
@posts.order(created_at: :desc)
```

Which will ordered by the `created_at` field — more resilient to unusual situations.